app/assets/v2/css/dashboard.css

```diff
@@ -75,6 +75,7 @@ body {
   display: flex;
   margin-top: 2em;
   margin-right: 8px;
+  padding-left: 8px !important;
```

Comment:
Can we do without using `!important` ?

Reply:
@thelostone-mc 
Thanks for the reminder! 😄